soc/arm64/bcm2711/Kconfig.defconfig

```diff
@@ -0,0 +1,17 @@
+# Copyright 2023 honglin leng <a909204013@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_BCM2711
+
+config SOC
+	default "bcm2711"
+
+config NUM_IRQS
+	int
+	default 260
+
+config SYS_CLOCK_HW_CYCLES_PER_SEC
+	int
+	default 25000000
```

Comment:
According to my experience, SYS_CLOCK_HW_CYCLES_PER_SEC could be  54000000

Reply:
CONFIG_TIMER_READS_ITS_FREQUENCY_AT_RUNTIME   enable it

Reply:
The SYS_CLOCK_HW_CYCLES_PER_SEC should be carefully set. I am not sure which Exception level the Raspberry Pi boots from, but I guess it is from EL1? That means we cannot set the `cntfreq_el0`, this register can only be set at the highest EL, it's likely set by firmware, so, that means we should make sure the actual value of `cntfreq_el0` is 25000000 or 54000000.